build.sbt

```diff
@@ -9,12 +9,14 @@ version := "0.9.1-SNAPSHOT"
 
 organization := "be.doeraene"
 
-scalaVersion := "2.11.7"
+scalaVersion := "2.11.8"
 
-crossScalaVersions := Seq("2.10.6", "2.11.7")
+crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.0-RC2")
 
 libraryDependencies +=
-  "org.scala-js" %%% "scalajs-dom" % "0.9.0"
+  "org.scala-js" %%% "scalajs-dom" % "0.9.1"
+
+resolvers += Resolver.sonatypeRepo("releases")
```

Comment:
It seems to be: https://repo1.maven.org/maven2/org/scala-js/scalajs-dom_sjs0.6_2.12.0-RC2/0.9.1/